src/lua/help_en_US.lua

```diff
@@ -168,7 +168,7 @@ Tarantool instruction that means <don’t execute
 every time I type Enter; wait until I type a
 special string called the "delimiter".>
 More in the Tarantool manual:
-http://tarantool.org/doc/book/administration.html#requests
+http://tarantool.io/en/doc/2.1/reference/reference_lua/console/#console-delimiter
```

Comment:
It would be painful to update these links with each new release and have no ability to cleanly cherry-pick changes downward (say, from 2.1 to 1.10). I propose to implement `https://tarantool.io/en/doc/latest/...` pages (or at least redirects) on a server side and just use them. The word 'latest' is common for urls on 'readthedocs', see at right botton corner at https://tarantool-python.readthedocs.io/en/latest/index.html for example.

Reply:
Decided to use a <version> placeholder and replace them with 2.1 / 1.10 as needed in a lua code.